edudoc/delegate-handbook/delegate-handbook.md

```diff
@@ -749,188 +742,146 @@ If an illegal puzzle is found during a round, any completed attempts may be repl
 
-The following is a general decision tree for handling incidents, and gives you a general idea of what to consider and what the outcome usually is for an incident. However, **this decision tree is always welcome to be overruled by the Delegate if they believe their decision is more fair.**
+The following is a general decision tree for handling incidents, and gives you a general idea of what to consider and what the outcome usually is for an incident. However, **this decision tree is always welcome to be overruled by the Delegate if they believe their decision is more fair**.
 
 ![](images/extra-decision-tree.png){.centered width=90%}
 
 #### Provisional Extras
 
 If you are ever unsure of whether it is fair to issue an extra, or do not have time to fully investigate the circumstances around an incident, you may always issue a provisional extra. A provisional extra is an extra that is not guaranteed to be accepted as the final result. Communicate to the competitor that you are granting a provisional extra and explain that whether the original time or the extra time will stand will be determined at a future time.
 
-If the provisional extra determines whether or not an individual advances to another round, you may choose to allow the affected competitor to compete provisionally as well. Inform them that their attempts for the next rounds are provisional, and may be thrown away depending on the final result of the other competitor’s provisional extra. Take caution when exercising this option, as it can often result in confusion for the affected competitor(s). Additionally, if this situation occurs more than once in a competition, rule all instances similarly in order to be fair to all competitors at that competition.
+If the provisional extra determines whether or not an individual advances to a subsequent round, you may choose to allow the affected competitor to compete provisionally as well. Inform them that their attempts for the subsequent rounds are provisional, and may be annulled depending on the final result of the other competitor's provisional extra. Take caution when exercising this option, as it can often result in confusion for the affected competitor(s). Additionally, if this situation occurs more than once in a competition, rule all instances similarly in order to be fair to all competitors at that competition.
```

Comment:
While "annulled" is the correct word, I'd recommend using simpler language like "may not be valid".

Reply:
Well, it should be an action, I think. How about "may be invalidated"?